### PR TITLE
Tree Doves

### PR DESCRIPTION
**Pull request by tomkettle1**
Adds Doves, and an improved render function to display them, that is much more expandable

### CODE REVIEW

**Comment by tomkettle1** [comment]
Ok, imma actually wait until my prev branch has been merged, cos this is tryna merge errything

**Comment by tomkettle1** [comment]
Fyi, master is currently broken, but this will fix it by making the age_days param for tree creation optional

File: Tree.cs [hunk 1]
```diff
@@ -24,8 +24,14 @@ class Tree
         //TODO: Make abstract
         public Tree(int age_days)
         {
+            Random r = new Random();
+
             Age_Days = age_days;
+            
+            if (r.Next(0, 50) == 0) { Has_Dove = true; }
```
Comment (PotatoHax101) [inline Tree.cs:31]:
```suggestion
            if (r.Next(0, 51) == 0) { Has_Dove = true; }
```

**Comment by tomkettle1** [comment]
(Note: remove all commits except the last two from the squash commit message)

**Pull request closed without merging**